Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.py

```diff
@@ -463,6 +463,7 @@ def search_group_members(default_base_dn, page_size):
     args = demisto.args()
     member_type = args.get('member-type')
     group_dn = args.get('group-dn')
+    time_limit = int(args.get('time_limit', 30))
```

Comment:
```suggestion
    time_limit = int(args.get('time_limit', 180))
```
lets make the the defaults the same